ttnn/cpp/ttnn/operations/experimental/transformer/nlp_create_qkv_heads_decode/device/nlp_create_qkv_heads_decode_device_operation.cpp

```diff
@@ -33,34 +33,43 @@ void NLPCreateHeadsDecodeDeviceOperation::validate(const std::vector<Tensor>& in
     TT_FATAL(input_shape[0] == 1, "Unsupported input shape");
     const auto QKV_memcfg = input_tensor.memory_config();
     if (input_tensor.is_sharded()) {
-        TT_FATAL(QKV_memcfg.memory_layout == TensorMemoryLayout::WIDTH_SHARDED, "Error");
+        TT_FATAL(QKV_memcfg.memory_layout == TensorMemoryLayout::WIDTH_SHARDED, "input tensor must be width sharded");
```

Comment:
For example you need to print current layout. It will help users a lot.

Reply:
Thanks for comments. Added more error info.